### PR TITLE
feat(stdlib)!: Convert unsafe `Wasm` functions to operators

### PR DESCRIPTION
**Pull request by spotandjake**
This pr changes the names of some of the Wasm Unsafe Runtime Operations such as `WasmI32.add` to be operators instead.

Currently Completed is `WasmF32` and `WasmF64`

Breaking: This is breaking because we are chaining the module apis.

This is a Draft because I still have todo `WasmI32` and `WasmI64`.

### CODE REVIEW

File: stdlib/runtime/numbers.gr [hunk 1]
```diff
@@ -234,10 +244,10 @@ let gcd32 = (x, y) => {
 
 @unsafe
 provide let reducedInteger = x => {
-  if (
-    WasmI64.gtS(x, WasmI64.shrS(_I32_MAX, 1N)) ||
-    WasmI64.ltS(x, WasmI64.shrS(_I32_MIN, 1N))
-  ) {
+  from WasmI64 use { (>>), (<), (>) }
+  // TODO(#1736): Remove the formatter ignore when parsing is fixed
+  //formatter-ignore
```
Comment (spotandjake) [inline stdlib/runtime/numbers.gr:249]: If #1736 gets patched before this is merged then this should be removed.

File: compiler/test/input/memoryBase/asserts.gr [hunk 2]
```diff
@@ -2,7 +2,6 @@
 module Asserts
 
 include "runtime/unsafe/wasmi32"
-
```
Comment (ospencer) [inline compiler/test/input/memoryBase/asserts.gr:5]: Can we have this space back?

File: package-lock.json [hunk 3]
```diff
@@ -19,11 +19,11 @@
     },
     "cli": {
       "name": "@grain/cli",
-      "version": "0.4.7",
+      "version": "0.5.13",
```
Comment (ospencer) [inline package-lock.json:22]: I don't know why these aren't up to date, but we shouldn't do these changes in this PR.

File: stdlib/rational.gr [hunk 4]
```diff
@@ -123,20 +123,21 @@ provide let (!=) = (x: Rational, y: Rational) => {
   !rationalsEqual(x, y)
 }
 
+from WasmI32 use { (<), (>), (<=), (>=) }
```
Comment (ospencer) [inline stdlib/rational.gr:126]: I'd like it a bit more if these were in the individual functions.

File: stdlib/runtime/atof/lemire.gr [hunk 5]
```diff
@@ -171,31 +163,34 @@ provide let computeFloat = (exponent: WasmI64, mantissa: WasmI64) => {
     match (lo == 0xFFFF_FFFF_FFFF_FFFFN && !(q >= -27N && q <= 55N)) {
       true => fpErr(),
       false => {
-        let upperbit = WasmI32.wrapI64(WasmI64.shrU(hi, 63N))
-        let mut mantissa = WasmI64.shrU(
-          hi,
+        from WasmI64 use { (>>>) }
```
Comment (ospencer) [inline stdlib/runtime/atof/lemire.gr:166]: Might as well just add this one to the rest on line 132.

File: package-lock.json (hunk 3 above)
Comment (spotandjake) [reply package-lock.json:22]: Yeah i accidently pushed that.

File: package-lock.json
Comment (ospencer) [inline package-lock.json:1]: These changes shouldn't be here.

Comment (spotandjake) [reply package-lock.json:1]: Fixed.

**Review by ospencer — APPROVED** [review]
I know this one was a huge undertaking! Truly excellent work getting this done 🥳

**Pull request merged**